src/Container/ApplicationFactory.php

```diff
@@ -162,15 +168,29 @@ private function injectRoutes(Application $app, ContainerInterface $container)
                 continue;
             }
 
-            $methods = (isset($spec['allowed_methods']) && is_array($spec['allowed_methods']))
-                ? $spec['allowed_methods']
-                : null;
+            $methods = Route::HTTP_METHOD_ANY;
```

Comment:
I was in doubt between the two, I eventually decided to use no-else style. But ok, I'll put `else` there.